### PR TITLE
feat: icons

### PR DESCRIPTION
**Pull request by bladicito**
https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/4

### CODE REVIEW

**Comment by bladicito** [comment]
failing because of typography.  After merging typo improvements into the master, merge master into this branch

File: globals.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+declare module '*.svg' {
```
Comment (lehmi11) [inline globals.d.ts]: What is the declaration for? Is it to turn the SVGs into React Components?

Comment (bladicito) [reply globals.d.ts]: yes, we needed to overwrite the behaviour to use it as a component and not as a string (path) to be used as an image src. Anyway, i removed it at the end and changed the approach to load the svgs.

File: components/icon/Icon.tsx [hunk 2]
```diff
@@ -10,40 +10,41 @@ import React, { useMemo } from 'react';
...
-import LocationIcon from './svg/location.svg';
-import LogoutIcon from './svg/logout.svg';
-import ProfileIcon from './svg/profile.svg';
-import RepostIcon from './svg/repost.svg';
-import SendIcon from './svg/send.svg';
-import SettingsIcon from './svg/settings.svg';
-import ShareIcon from './svg/share.svg';
-import TimeIcon from './svg/time.svg';
-import UploadIcon from './svg/upload.svg';
+import mumbleIcon from './svg/mumble';
```
Comment (lehmi11) [inline components/icon/Icon.tsx:13]: Why did you change the names and do not return React components anymore? Are there any benefits with this approach?

Comment (bladicito) [reply components/icon/Icon.tsx:13]: I was not able to get rid of the type errors. I invested a lot of time trying to fix it but then I did not want to lose more time. We can try again later on but the functionality would be exactly the same. 
PRO here. no needed dependency nor extra webpack config.

**Review by lehmi11 — APPROVED** [review]
Only some questions about implementation, otherwise LGTM

File: components/icon/Icon.tsx (hunk 2 above)
Comment (lehmi11) [reply components/icon/Icon.tsx:13]: That is good to me. Thx for the explanation
